Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -115,6 +115,14 @@ theorem two_mul_card_alternatingGroup [Nontrivial α] :
     2 * card (alternatingGroup α) = card (Perm α) := by
   simp only [← Nat.card_eq_fintype_card, ← alternatingGroup.index_eq_two (α := α), index_mul_card]
 
+theorem card_alternatingGroup [Nontrivial α] :
+    card (alternatingGroup α) = (card α).factorial / 2 := by
```

Comment:
Can these results be stated in terms of `Nat.card`, to match the rest of the group theory library? Or at least add `Nat.card` versions (like `Nat.card_perm`)?

Reply:
They can, they were just in terms of Fintype, so I went along. Should we duplicate?

Reply:
Sure, go for it.